versioned_hdf5/backend.py

```diff
@@ -188,14 +192,17 @@ def write_dataset(
                     hashed_slice = hashtable[data_hash]
                     slices[data_slice] = hashed_slice
 
-                    _verify_new_chunk_reuse(
-                        raw_dataset=ds,
-                        new_data=data,
-                        data_hash=data_hash,
-                        hashed_slice=hashed_slice,
-                        chunk_being_written=data_s,
-                        slices_to_write=slices_to_write,
-                    )
+                    if os.environ.get(
+                        "ENABLE_CHUNK_REUSE_VALIDATION", "false"
+                    ).lower() in ("1", "true"):
```

Comment:
Can we move this environment variable handling out of the loop?

Reply:
100% - done.